client/src/composables/useInvocationGraph.ts

```diff
@@ -169,7 +173,7 @@ export function useInvocationGraph(
                     invocationStepSummary = stepsJobsSummary.find((stepJobSummary: StepJobSummary) => {
                         if (stepJobSummary.model === "ImplicitCollectionJobs") {
                             return stepJobSummary.id === invocationStep.implicit_collection_jobs_id;
-                        } else {
+                        } else if (stepJobSummary.model === "Job") {
                             return stepJobSummary.id === invocationStep.job_id;
```

Comment:
@jmchilton As you can see in this https://github.com/galaxyproject/galaxy/pull/17708/commits/78f653c5141c5be37a43adb0b5fc49325a7ee8f1 commit, I needed to use `LegacyWorkflowInvocationElementView` because right here, we need the `job_id` to associate the stepJobSummary with it. 
Of course, overall, I need to rebase and readjust these changes, but is this the right approach as far as the view is concerned?